_posts/2016-05-06-elasticsearch-module_3.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: Elasticsearch - Realizando la primera búsqueda
+usernames: [maira1001001, rosariosm]
+tags: [elasticsearch, bulk API, search API]
+---
+
+En este módulo veremos cómo crear un índice, agregarle un conjunto de datos provenientes de un JSON válido y realizar una búsqueda simple mediante la API de búsqueda.
+
+
+
+## Creando un índice
+
+En este módulo trabajaremos con ejemplos de artículos de diario. Antes de comenzar necesitamos crear el índice **article_index** para almacernarlos. 
+
+{% highlight bash %}
+$ curl -XPUT 'http://localhost:9200/article_index'
+{% endhighlight %}
+
+Para visualizar el índice creado, escriba en consola la siguiente consulta:
+
+{% highlight bash %}
+$ curl -XGET 'http://localhost:9200/article_index?pretty'
+{% endhighlight %}
+
+El resultado será lo siguiente:
+
+{% highlight bash %}
+{
+  "article_index" : {
+    "aliases" : { },
+    "mappings" : { },
+    "settings" : {
+      "index" : {
+        "creation_date" : "1462807670938",
+        "uuid" : "E84HLb0kSDC5QzFlni8R8g",
+        "number_of_replicas" : "1",
+        "number_of_shards" : "5",
+        "version" : {
+          "created" : "1040599"
+        }
+      }
+    },
+    "warmers" : { }
+  }
+}
+{% endhighlight %}
+
+Como podrá observar, el índice se creo con algunas características por defecto y otras por configurar.
+
+* **"aliases"**: Define un conjunto de aliases para el índice.
```

Comment:
En estos también usaría `monospaced` para los nombres que son propiedades del JSON.

De hecho, en general (así ya no insisto con esto) me parece que resulta más claro cuando se habla de referencias a código/elementos de un archivo/propiedades de un objeto/etc, usar `monospaced` para indicar eso, distinguiéndolo de las referencias a palabras *in English* o **importantes**.. Si manejamos un formato claro y uniforme agiliza a la identificación rápida de los contenidos..